test_data/yast/skip_registration/default_repos.yaml

```diff
@@ -15,10 +15,8 @@ repos:
     uri: cd:/\?devices=/dev/disk/by-id/scsi-0QEMU_QEMU_CD-ROM_cd0
     enabled: No
     filter: name
-  - name: SLES
-    nr: 3
+  - name: 'SLES-%VERSION%'
```

Comment:
Please, split this change to the separate commit.

Reply:
doesn't look useful for this use case, after version we need [15.3-0](https://openqa.suse.de/tests/5421445#step/validate_repos/12) which is not in vars.json. I dropped the commit for now.